pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/KitUtil.java

```diff
@@ -120,14 +121,15 @@ public class KitUtil {
     private static final String EASYPOST_FAILURE_STATUS = "failure";
     private static final String EASYPOST_RETURN_SENDER_STATUS = "return_to_sender";
     private static final String EASYPOST_ERROR_STATUS = "error";
-    public static final String PECGS_RESEARCH = "PECGS_RESEARCH";
 
-    public static void createLabel(List<KitRequestCreateLabel> kitsLabelTriggered) {
+    public static void createLabel(List<KitRequestCreateLabel> kitsLabelTriggered, EasyPostUtil easyPostUtil) {
```

Comment:
We need to create a new EasyPostUtil because some studies use different easypost accounts. The only change I made was for instances when we want to test to enable us to pass a mock easypost

Reply:
So for that passed-in EasyPostUtil, the instance/realm does not matter? If so, you might want to add some JavaDoc, since otherwise other readers may have the same question.

Reply:
So the reason we make different ones for each realm is to pass the right apikey and secret to real easypost. For the mock one we just have to mock everything and not use the real easypost.